src/screens/Bridge/Exchange-CrossChain/OfferCard.js

```diff
@@ -97,11 +97,14 @@ function OfferCard({
   const [offerInfo, setOfferInfo] = React.useState(null);
 
   React.useEffect(() => {
-    async function call() {
+    let mounted = true;
+    (async () => {
```

Comment:
The use of `mounted` makes more sense here, but I would still recommend `useState` to have a global `mounted` variable

Also please put the async function into a variable, and then call that variable - it looks a bit messy like this

Reply:
Here main purpose of `mounted` is cleanup async function every time. `useState` is not used in this case because useState is the main cause of memory leaks. 

-> And (async() => {})() this is Immediately Invoked Function Expression
https://developer.mozilla.org/en-US/docs/Glossary/IIFE

Reply:
Same comment about the mounted/clean up here - you're set mounted as true on line 100, and then immediately using it on line 103. This isn't inside an async function or anything so there is no delay between this (less than a millisecond) and it will always be true, regardless of the clean up function. It's better to use the hook here.

Re IIFE: Yeah I'm aware, its probably just personal preference but I tend to avoid using them as I find them to look messier and more difficult to read when they become more bloated. Also since IIFEs in React only are ever used inside hooks (I can't think of any other reasons to use them), I think having consistency is better

Reply:
In this `mounted` managed with hooks